src/views/level0/Level0-1.vue

```diff
@@ -13,11 +13,12 @@
         class="x-input"
         @valid="forms.vset.input.name"
         v-model="forms.input.name"
-        placeholder="Enter your name"
+        placeholder="Enter your Telegram or Discord handle"
         required
       />
+      <button commons @click="goNext" class="mt-2" :disabled="!forms.vget.input.form">next</button>
+      <h2 class="teko-subtitle mt-1p5" style="cursor: pointer;" @click="$router.push('/intro')">watch intro again</h2>
```

Comment:
I talked to Marko and he said the `watch intro again` link should have the same style as the Exit and Help buttons.

Reply:
oh okay!

Reply:
<img width="1440" alt="Screenshot 2021-01-24 at 15 45 18" src="https://user-images.githubusercontent.com/44097430/105633824-2c1c0200-5e5b-11eb-9609-60712caf846f.png">
then is it ok like that?

Reply:
yeah, I think so.